### ISSUE TITLE
Show tooltips for the buttons in the view mode switcher?

### ISSUE THREAD

**Issue by nekohayo** [getting-things-gnome/gtg#325]
Wondering if it's good practice to have Gtk Tooltips showing up when the mouse is over the "Open | Actionable | Closed" buttons? If that's not considered weird (or annoying for users), then we could have that UI part provide reminders of what the distinction are, what are the criteria of things being shown in each mode... that would make it kind of a self-documented user interface to complement @vansia43's user manual documentation work.

**Comment by vansia43** [comment]
@nekohayo - Perhaps, something like:

- **Open**: "View all tasks not marked as closed" or "View all open tasks"

- **Actionable**: "View tasks you can start right now" 

- **Closed**: "View tasks marked as done or dismissed"

I feel like with Actionable, if you start listing the criteria, the tooltip could get really long and then be annoying. But I think the simple statement of  "start right now" implies that the task doesn't have anything blocking it. We go into this criteria, in-depth, in the sample tasks and the user manual. A quick reminder seems the least intrusive to me.

**Comment by nekohayo** [comment]
Hesitating as to whether this is worth it now, as it's pretty well documented in the user manual and the "first run" starter tasks, especially when my "translations" branch will be merged to master as it overhauls the tutorial tasks...

**Comment by vansia43** [comment]
@nekohayo after viewing the starter tasks in the translations branch, I would say maybe not to do this. This is very well documented there and in the manual and for someone toggling between different views, a tooltip could get in the way.

**Comment by nekohayo** [comment]
Agreed!

**Comment by nekohayo** [comment]
I have found a new, better reason to want this now: universal accessibility.

For blind users relying on screen readers apps like Orca, the current tabs will be read just as:

> "Open. Page tab."

> "Actionable. Page tab."

> "Closed. Page tab."

That doesn't tell the blind user anything, i.e. what will actually happen if they toggle those tabs, because they won't be able to see the direct result.

As such, explicit tooltips would be helpful for a11y there.
